search-app/src/cmr/search/services/parameters/converters/geometry.clj

```diff
@@ -1,11 +1,13 @@
 (ns cmr.search.services.parameters.converters.geometry
   "Contains parameter converters for shapefile parameter"
   (:require
+   [cmr.common.log :refer (debug)]
```

Comment:
nit: I think we started using vectors for refer. i.e. : `[cmr.common.log :refer [debug]]`

Reply:
fixed